cime_config/acme/machines/python.case.test

```diff
@@ -4,6 +4,10 @@
  This is the system test submit script for CIME
 
 """
+import os
```

Comment:
PBS systems do not default to the cwd on batch submission so you need to change to that directory.